### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by wychoi44** [google/uis-rnn#63]
## Describe the question

The d-vector embeddings are trained on TIMIT corpus (EER=4.17%) using the github(https://github.com/HarryVolek/PyTorch_Speaker_Verification).
For UIS-RNN experiments, I trained on Switchboard corpus, and tested on Callhome American English dataset as mentioned in the paper, "Fully supervised speaker diarization."
However, DER was 30.1%, which is far from the results in the paper(11.7%).
DER on the toy data is pretty good(0.4%). 
I used the same configurations provided. What is the problem?

## My background

Have I read the `README.md` file?
* yes

Have I searched for similar questions from closed issues?
* yes

Have I tried to find the answers in the paper *Fully Supervised Speaker Diarization*?
* yes

Have I tried to find the answers in the reference *Speaker Diarization with LSTM*?
* yes

Have I tried to find the answers in the reference *Generalized End-to-End Loss for Speaker Verification*?
* yes
